app/jinja_filters.py

```diff
@@ -541,7 +541,6 @@ def __init__(
         edit_link_text: str,
         edit_link_aria_label: str,
     ) -> None:
-        self.rowTitle = strip_tags(question["title"])
```

Comment:
Why is the question title being removed...?. If you have a question with multiple answers then you will never see the question title with this change.